geofencing_zones.json

```diff
@@ -32,116 +32,127 @@
-                        "description": "Start time of the geofencing zone in POSIX time.",
-                        "type": "number",
-                        "minimum": 1450155600
-                      },
-                      "end": {
-                        "description": "End time of the geofencing zone in POSIX time.",
-                        "type": "number",
-                        "minimum": 1450155600
-                      },
-                      "rules": {
-                        "description": "Array that contains one object per rule.",
-                        "type": "array",
-                        "items": {
-                          "vehicle_type_id": {
-                            "description": "Array of vehicle type IDs for which these restrictions apply.",
-                            "type": "array"
-                          },
-                          "ride_allowed": {
-                            "description": "Is the undocked ride allowed to stat and end in this zone?",
-                            "type": "boolean"
-                          },
-                          "ride_through_allowed": {
-                            "description": "Is the ride allowed to travel through this zone?",
-                            "type": "boolean"
-                          },
-                          "maximum_speed_kph": {
-                            "description": "What is the maximum speed allowed, in kilometers per hour?",
-                            "type": "number",
-                            "minimum": 0
+                      "description": "Describing travel allowances and limitations.",
+                      "type": "object",
+                      "properties": {
+                        "name": {
+                          "description": "Public name of the geofencing zone.",
+                          "type": "string"
+                        },
+                        "start": {
+                          "description": "Start time of the geofencing zone in POSIX time.",
+                          "type": "number",
+                          "minimum": 1450155600
+                        },
+                        "end": {
+                          "description": "End time of the geofencing zone in POSIX time.",
+                          "type": "number",
+                          "minimum": 1450155600
+                        },
+                        "rules": {
+                          "description": "Array that contains one object per rule.",
+                          "type": "array",
+                          "items": {
+                            "type":"object",
+                            "properties": {
+                              "vehicle_type_id": {
+                                "type": "array",
+                                "description": "Array of vehicle type IDs for which these restrictions apply.",
+                                "items": {"type": "string"}
```

Comment:
This change seems to match the intent of this property based on the description.